packages/@cdklabs/cdk-cicd-wrapper/src/code-pipeline/AppStage.ts

```diff
@@ -50,33 +45,10 @@ export class AppStage extends cdk.Stage {
 
       const encryptionStack = context.get(GlobalResources.ENCRYPTION)!;
 
-      new LogRetentionRoleStack(this, `${applicationName}LogRetentionRoleStack`, {
```

Comment:
Please provide a note to cleanup manually this already deployed stack as this wont be cleaned up automatically when re-deploying.

Reply:
done see Migration.md